loom/common/assets/assets.cpp

```diff
@@ -1161,6 +1162,17 @@ void loom_asset_reloadAll()
     }
 }
 
+void loom_asset_clear()
+{
+    utHashTableIterator<utHashTable<utHashedString, loom_asset_t *> > assetIterator(gAssetHash);
+    while (assetIterator.hasMoreElements())
+    {
+        utHashedString key = assetIterator.peekNextKey();
+        lmDelete(NULL, assetIterator.peekNextValue());
```

Comment:
The comment says this deletes names but won't it implicitly delete the loaded asset too?